sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -1287,13 +1291,15 @@ static Task WritePartitionAsync(Response<BlobDownloadInfo> response, Stream dest
         public virtual Operation<long> StartCopyFromUri(
             Uri source,
             Metadata metadata = default,
+            AccessTierOptional? accessTier = default,
```

Comment:
We don't want a type named `AccessTierOptional` - it doesn't match the naming conventions for .NET types.  What is this type and how does it differ from `AccessTier`?

Reply:
Apparently this is a remnant of a workaround for the swagger being in a partially updated state.  You can fix this by:

- Deleting the section https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/swagger/readme.md#make-accesstier-unique
- Pulling the latest changes from master
- Regenerating according to the instructions at https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/CONTRIBUTING.md#code-generation
- `AccessTierOptional` will just become `AccessTier` and you can replace it anywhere the compiler complains